sdk/attestation/azure-security-attestation/samples/policy/reset_sealed_policy.cpp

```diff
@@ -56,15 +56,13 @@ int main()
     clientOptions.TokenValidationOptions.TimeValidationSlack = 10s;
 
     // create client
-    auto const credential = std::make_shared<Azure::Identity::ClientSecretCredential>(
-        GetEnvHelper::GetEnv("AZURE_TENANT_ID"),
-        GetEnvHelper::GetEnv("AZURE_CLIENT_ID"),
-        GetEnvHelper::GetEnv("AZURE_CLIENT_SECRET"));
+    auto const credential = std::make_shared<Azure::Identity::EnvironmentCredential>();
```

Comment:
Just to make  sure I am not misunderstanding something, `EnvironmentCredential` just makes setting/passing the required env variables to create the credential easier, correct? And it doesn't get rid of the need to set these env variables on the user's machine, all together, right?